frontend/src/app/offers/create-offer/create-offer.component.html

```diff
@@ -0,0 +1,299 @@
+<div *ngIf="isAdmin; else hasOrg">
+  <p>Administrator nie może tworzyć nowych ofert.</p>
+</div>
+<ng-template #hasOrg>
+  <ng-container *ngIf="!hasOrganization; else offerForm">
+  <p>Aby dodać ofertę musisz posiadać organizację.
+    <a href="{{ djangoRoot }}/organizations/create">Możesz ją założyć tutaj</a>
+  </p>
+  </ng-container>
+</ng-template>
+<ng-template #offerForm>
+<h2>{{ createOrEdit }} oferty:</h2>
```

Comment:
Delete createOrEdit. You can use 'edit' variable from component to create a ternary operator
{{ edit ? "Edycja" : "Tworzenie" }} and decide on the string displayed